# Current flying state of the drone

Header header

string flying_state